File: labs-training-master/java/solid/solid-java-1/src/main/java/es/emergya/labs/training/solid_java_1/ocp/excercise_00/PizzaIngredientsPackage.java
package es.emergya.labs.training.solid_java_1.ocp.excercise_00;

public class PizzaIngredientsPackage implements IngredientsInterface {

	private static String message = "Baking pizza with %.2fKg of cheese, %.2fKg of veggies and %.2fKg of meat.";
	
	private Float cheeseKg;
	private Float veggiesKg;
	private Float meatKg;

	public PizzaIngredientsPackage(Float cheeseKg, Float veggiesKg, Float meatKg) {
		super();
		this.cheeseKg = cheeseKg;
		this.veggiesKg = veggiesKg;
		this.meatKg = meatKg;
	}

	void bake() {

		System.out.println(String.format(message, cheeseKg, veggiesKg, meatKg));
	}

	public void makeFood(){
		bake();
	}

}

File: labs-training-master/SolidPythonResuelto/main/ocp/PizzaIngredientsPackage.py
from IngredientsPackageInterface import IngredientsPackageInterface

class PizzaIngredientsPackage(IngredientsPackageInterface) :
    def __init__(self, cheeseKg:str,  veggiesKg:str,  meatKg:str) :
        self.cheeseKg = cheeseKg
        self.veggiesKg = veggiesKg
        self.meatKg = meatKg
        self.message:str = "Baking pizza with {0}Kg of cheese,  {1}Kg of veggies and {2}kg of meat.".format(str(cheeseKg),str(veggiesKg),str(meatKg))
    def bake(self) -> None :
        print(self.message)
    
    def makeFood(self) -> None:
        self.bake()